scrapy_poet/middleware.py

```diff
@@ -58,7 +62,7 @@ def process_request(self, request: Request, spider: Spider):
         if self.injector.is_scrapy_response_required(request):
             return
 
-        spider.logger.debug(f'Skipping download of {request}')
+        logger.debug(f"Using DummyResponse instead of downloading {request}")
```

Comment:
As suggested, in case you liked it:
```suggestion
        logger.debug(f"DummyResponse in use. Scrapy downloader won't fetch {request}")
```